docs/ingest-via-opentelemetry-api-endpoint-fdc78af.md

```diff
@@ -70,8 +70,75 @@ OpenTelemetry support in SAP Cloud Logging needs to be enabled with a service in
 
     You can use the endpoint and credentials to ship OpenTelemetry signals retrieved by one of the following instrumentation options: OpenTelemetry SDK, OpenTelemetry Agent, or OpenTelemetry Collector. See [OpenTelemetry documentation on instrumentation](https://opentelemetry.io/docs/concepts/instrumentation/) for more details.
 
+### Automated OpenTelemetry Exporter Configuration for Java and node.JS Applications in the Cloud Foundry Environment
 
+When you follow the procedure outlined above all newly created service bindings will contain the credentials required to configure OpenTelemetry instrumentations.
+The Cloud Logging team provides special instrumentation for Java and node.JS applications, that utilize OpenTelemetry instrumentation.
```

Comment:
```suggestion
After following the procedure, newly created service bindings will contain required credentials.
Below we describe simplified way of using them to instrument Java and node.JS applications harmonized with other SAP solutions.
```

Reply:
This is a new section. Just referring to "the procedure" might be misleading, since there is no procedure in this section.